docs/content/showcase.md

```diff
@@ -66,4 +66,9 @@ starters:
     url: https://shadcn-docs.nuxt.dev/
     image: https://github.com/ZTL-UwU/shadcn-docs-nuxt/assets/49156174/b317288d-092d-4656-af5e-54034351daca
 
+ - title: Origin UI Vue
+    description: Beautiful UI components built with Tailwind CSS and Radix Vue
```

Comment:
The starter was added as [shadcn-docs](https://github.com/unovue/radix-vue/pull/1454/files#diff-e2ca8600e9ed4ee3871c2562f9710a0e3aec1f5ec6c8916c4d54a3734c7ab011R64-R65) previously. Maybe you can update the title and description 😁

Reply:
Updated, thanks for letting me know